package.json

```diff
@@ -76,6 +77,7 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^22.0.4",
     "jest-cli": "^22.0.4",
+    "markdown-loader-jest": "^0.1.1",
```

Comment:
Need [`markdown-loader-jest`](https://github.com/peerigon/markdown-loader) in order to import `README.md` files for stories.